gef.py

```diff
@@ -5645,7 +5645,19 @@ def search_pattern_by_address(self, pattern, start_address, end_address):
             else:
                 chunk_size = step
 
-            mem = read_memory(chunk_addr, chunk_size)
+            try:
+                mem = read_memory(chunk_addr, chunk_size)
+            except gdb.error as e:
+                estr = str(e)
+                if estr.startswith("Cannot access memory "):
+                    #
+                    # This is a special case.
```

Comment:
Add a brief description, Something like, "We can sometimes fail to read memory even if it's +r"

Reply:
Working on this.

Reply:
Resolved.